source/docs/software/wpilib-tools/robot-simulation/introduction.rst

```diff
@@ -33,12 +33,15 @@ Often a team may want to test their code without having an actual robot availabl
 
       **Running Robot Simulation**
 
-      Basic robot simulation can be run using VS Code. This can be done without using any commands by using VS Code's command palette.
+      Basic robot simulation can be run using VS Code. This can be done by using VS Code's command palette :guilabel:`WPILib: Simulate Robot Code` as shown below.
 
       .. image:: images/vscode-run-simulation.png
          :alt: Running robot simulation through VS Code
 
-      Your console output in Visual Studio Code should look like the below. However, teams probably will want to actually *test* their code versus just running the simulation. This can be done using :doc:`WPILib's Simulation GUI <simulation-gui>`.
+      .. image:: images/vscode-pick-extension.png
+         :alt: Picking Sim GUI simulation extension
+
+      By default the :guilabel:`Sim GUI` option will be selected by default. This will launch the :doc:`Simulation GUI <simulation-gui>`. You can also launch simulation without a GUI by unchecking :guilabel:`Sim GUI` in which case your console output in Visual Studio Code should look like the below.
```

Comment:
```suggestion
     The :guilabel:`Sim GUI` option will be selected by default. This will launch the :doc:`Simulation GUI <simulation-gui>`. You can also launch simulation without a GUI by unchecking :guilabel:`Sim GUI` in which case your console output in Visual Studio Code should look like the below.
```